diff -ur flyspray-0.9.9.5.1/setup/index.php flyspray-0.9.9.5.1.new/setup/index.php
--- flyspray-0.9.9.5.1/setup/index.php	2008-03-07 18:55:22.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/index.php	2008-07-10 06:11:35.000000000 +0000
@@ -12,9 +12,9 @@
 ini_set('memory_limit', '64M');
 
 
-if (is_readable ('../flyspray.conf.php') && count(parse_ini_file('../flyspray.conf.php')) > 0)
+if (is_readable ('/etc/webapps/flyspray/flyspray.conf.php') && count(parse_ini_file('/etc/webapps/flyspray/flyspray.conf.php')) > 0)
 {
-   die('Flyspray already installed. Use the <a href="upgrade.php">upgrader</a> to upgrade your Flyspray, or delete flyspray.conf.php to run setup.
+   die('Flyspray already installed. Use the <a href="upgrade.php">upgrader</a> to upgrade your Flyspray, or delete /etc/webapps/flyspray/flyspray.conf.php to run setup.
         You can *not* use the setup on an existing database.');
 }
 
@@ -88,7 +88,7 @@
    function Setup()
    {
       // Look for ADOdb
-      $this->mAdodbPath         = APPLICATION_PATH . '/adodb/adodb.inc.php';
+      $this->mAdodbPath         = '/usr/share/php/adodb/adodb.inc.php';
       $this->mProductName       = 'Flyspray';
       $this->mMinPasswordLength	= 8;
 
@@ -806,7 +806,7 @@
 
       $config_text = $config_intro . implode( "\n", $config );
 
-      if (is_writable('../flyspray.conf.php') && ($fp = fopen('../flyspray.conf.php', "wb")))
+      if (is_writable('/etc/webapps/flyspray/flyspray.conf.php') && ($fp = fopen('/etc/webapps/flyspray/flyspray.conf.php', "wb")))
       {
          fputs($fp, $config_text, strlen($config_text));
          fclose($fp);
Tylko w flyspray-0.9.9.5.1.new/setup: index.php~
diff -ur flyspray-0.9.9.5.1/setup/templates/complete_install.tpl flyspray-0.9.9.5.1.new/setup/templates/complete_install.tpl
--- flyspray-0.9.9.5.1/setup/templates/complete_install.tpl	2007-02-04 07:11:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/complete_install.tpl	2008-07-04 12:10:36.000000000 +0000
@@ -21,7 +21,7 @@
 							The configuration file is not writeable. You will have to upload the following
 							code manually. Click in the textarea to highlight all of the code. Copy and
 							paste the contents into the flyspray.conf.php file available in the base of
-							{$product_name} installation.
+							{$product_name} (/etc/webapps/flyspray).
 						</td>
 					</tr>
 					<tr>
@@ -37,10 +37,10 @@
 				if (!$config_writeable)
 				{
 				?>
-				<h3>flyspray.conf.php NOT writeable</h3>
+				<h3>flyspray.conf NOT writeable</h3>
 				<p>
 					To complete setup, copy and paste the contents of the textarea box into flyspray.conf.php
-					This file resides in the base of your {$product_name} installation.
+					This file resides in the base of your {$product_name} installation (/etc/webapps/flyspray).
 				</p>
 				<?php
 				}
@@ -72,4 +72,4 @@
 				</div>
 			</form>
 			</div><!-- end of right -->
-			<div class="clr"></div>
\ Brak znaku nowej linii na końcu pliku
+			<div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: complete_install.tpl~
diff -ur flyspray-0.9.9.5.1/setup/templates/license.tpl flyspray-0.9.9.5.1.new/setup/templates/license.tpl
--- flyspray-0.9.9.5.1/setup/templates/license.tpl	2007-02-04 07:11:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/license.tpl	2008-07-04 12:13:11.000000000 +0000
@@ -11,7 +11,7 @@
          <div class="clr"></div>
 
          <div class="formBlock" style="width:470px;position:relative;">
-            <iframe src="../docs/licences/gnu_lgpl.html" class="license" width="450" frameborder="0" scrolling="auto"></iframe>
+            <iframe src="licences/gnu_lgpl.html" class="license" width="450" frameborder="0" scrolling="auto"></iframe>
          </div>
 
          <div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: license.tpl~
diff -ur flyspray-0.9.9.5.1/setup/templates/pre_install.tpl flyspray-0.9.9.5.1.new/setup/templates/pre_install.tpl
--- flyspray-0.9.9.5.1/setup/templates/pre_install.tpl	2007-03-24 13:26:36.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/pre_install.tpl	2008-07-04 12:12:30.000000000 +0000
@@ -38,7 +38,7 @@
                 </p>
                 <?php if (!$sapiStatus): ?>
                 <p><strong>CGI server API is not supported</strong>. Consider upgrading to FastCGI, otherwise you have to add
-                <code>force_baseurl = "http://yourflyspray/"</code> manually to flyspray.conf.php after setup.
+                <code>force_baseurl = "http://yourflyspray/"</code> manually to flyspray.conf after setup.
                 </p>
                 <?php endif; ?>
 			</div>
@@ -69,7 +69,7 @@
 			<div class="installBlock">
 				<table class="formBlock">
 				<tr>
-					<td valign="top">../flyspray.conf.php</td>
+					<td valign="top">../flyspray.conf</td>
 					<td align="left"><b>{!$config_output}</b></td>
 					<td>&nbsp;</td>
 				</tr>
@@ -93,12 +93,12 @@
 				</p>
 				<?php if (!$config_status): ?>
 				<p>
-				The installer has detected that the <strong>flyspray.conf.php</strong> file is not
+				The installer has detected that the <strong>flyspray.conf</strong> file is not
 				writeable. Please make it writeable by the web-server user or world writeable to
 				proceed with the setup. Alternatively if you wish to proceed, the installer will
 				make available the contents of the configuration file at the end of the setup. You
 				will then have to manually copy and paste the contents into the configuration file
-				located at <strong><?php echo APPLICATION_PATH . DIRECTORY_SEPARATOR . 'flyspray.conf.php'; ?></strong>.
+				located at <strong>'/etc/webapps/flyspray/flyspray.conf</strong>.
 				</p>
 				<?php endif; ?>
 			</div>
@@ -124,4 +124,4 @@
 			</div>
 			<div class="clr"></div>
 			</div><!-- end of right -->
-			<div class="clr"></div>
\ Brak znaku nowej linii na końcu pliku
+			<div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: pre_install.tpl~
