diff -urN flyspray-0.9.6_oryg/flyspray.conf.php flyspray-0.9.6/flyspray.conf.php
--- flyspray-0.9.6_oryg/flyspray.conf.php	2004-09-14 13:19:24.000000000 +0200
+++ flyspray-0.9.6/flyspray.conf.php	2004-10-13 19:15:02.902568288 +0200
@@ -6,9 +6,9 @@
 ; a web browser, and editing header.php to point to wherever you put this file.
 
 [general]
-basedir = "/var/www/flyspray/"                ; Location of your Flyspray installation
+basedir = "/usr/share/flyspray/"                ; Location of your Flyspray installation
 cookiesalt = "4t"                             ; Randomisation value for cookie encoding
-adodbpath = "/usr/share/adodb/adodb.inc.php"  ; Path to the main ADODB include file
+adodbpath = "/usr/share/pear/adodb/adodb.inc.php"  ; Path to the main ADODB include file
 output_buffering = "on"                     ; Available options: "off", "on" and "gzip"
 
 [database]
