diff -ur flyspray-0.9.9.5.1/includes/class.database.php flyspray-0.9.9.5.1.new/includes/class.database.php
--- flyspray-0.9.9.5.1/includes/class.database.php	2008-03-17 17:41:16.000000000 +0000
+++ flyspray-0.9.9.5.1.new/includes/class.database.php	2008-07-10 07:18:07.000000000 +0000
@@ -16,7 +16,7 @@
     die('Do not access this file directly.');
 }
 
-require_once dirname(dirname(__FILE__)) . '/adodb/adodb.inc.php';
+require_once '/usr/share/php/adodb/adodb.inc.php';
 
 class Database
 {
@@ -53,7 +53,7 @@
         if(!is_array($conf) || extract($conf, EXTR_REFS|EXTR_SKIP) < 5) {
 
             die( 'Flyspray was unable to connect to the database. '
-                 .'Check your settings in flyspray.conf.php');
+                 .'Check your settings in flyspray.conf');
         }
 
        $this->dbOpen($dbhost, $dbuser, $dbpass, $dbname, $dbtype, isset($dbprefix) ? $dbprefix : '');
@@ -82,7 +82,7 @@
         if ($this->dblink === false || (!empty($this->dbprefix) && !preg_match('/^[a-z][a-z0-9_]+$/i', $this->dbprefix))) {
 
             die('Flyspray was unable to connect to the database. '
-               .'Check your settings in flyspray.conf.php');
+               .'Check your settings in flyspray.conf');
         }
             $this->dblink->SetFetchMode(ADODB_FETCH_BOTH);
 
Tylko w flyspray-0.9.9.5.1.new/includes: class.database.php~
diff -ur flyspray-0.9.9.5.1/includes/class.flyspray.php flyspray-0.9.9.5.1.new/includes/class.flyspray.php
--- flyspray-0.9.9.5.1/includes/class.flyspray.php	2008-03-24 16:20:56.000000000 +0000
+++ flyspray-0.9.9.5.1.new/includes/class.flyspray.php	2008-07-10 07:18:53.000000000 +0000
@@ -113,7 +113,7 @@
 
     function get_config_path($basedir = BASEDIR)
     {
-        $cfile = $basedir . '/flyspray.conf.php';
+        $cfile = '/etc/webapps/flyspray/flyspray.conf';
         if (is_readable($hostconfig = sprintf('%s/%s.conf.php', $basedir, $_SERVER['SERVER_NAME']))) {
             $cfile = $hostconfig;
         }
Tylko w flyspray-0.9.9.5.1.new/includes: class.flyspray.php~
diff -ur flyspray-0.9.9.5.1/includes/constants.inc.php flyspray-0.9.9.5.1.new/includes/constants.inc.php
--- flyspray-0.9.9.5.1/includes/constants.inc.php	2008-02-22 21:07:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/includes/constants.inc.php	2008-07-10 07:20:17.000000000 +0000
@@ -8,7 +8,7 @@
 
 define('BASEDIR', dirname(dirname(__FILE__)));
 
-// Change this line if you move flyspray.conf.php elsewhere
+// Change this line if you move flyspray.conf elsewhere
 $conf = @parse_ini_file(Flyspray::get_config_path(), true);
 
 // $baseurl
@@ -83,4 +83,4 @@
 //define('JABBER_DEBUG', true);
 //define('JABBER_DEBUG_FILE''/path/to/my/debug/file');
 //define('FS_MAIL_LOGFILE', BASEDIR . '/logs/maillog.txt');
-?>
\ Brak znaku nowej linii na końcu pliku
+?>
Tylko w flyspray-0.9.9.5.1.new/includes: constants.inc.php~
diff -ur flyspray-0.9.9.5.1/setup/exportdb.php flyspray-0.9.9.5.1.new/setup/exportdb.php
--- flyspray-0.9.9.5.1/setup/exportdb.php	2007-02-04 07:11:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/exportdb.php	2008-07-10 07:19:56.000000000 +0000
@@ -3,10 +3,10 @@
 error_reporting(E_ALL);
  
  
-require_once '../adodb/adodb.inc.php';
-require_once '../adodb/adodb-xmlschema03.inc.php';
+require_once '/usr/share/php/adodb/adodb.inc.php';
+require_once '/usr/share/php/adodb/adodb-xmlschema03.inc.php';
  
-$conf    = @parse_ini_file('../flyspray.conf.php', true) or die('Cannot open config file.');
+$conf    = @parse_ini_file('/etc/webapps/flyspray/flyspray.conf', true) or die('Cannot open config file.');
  
 /* Start by creating a normal ADODB connection.
  */
@@ -26,4 +26,4 @@
  
 file_put_contents('flyspray-schema.xml', $data);
  
-?>
\ Brak znaku nowej linii na końcu pliku
+?>
Tylko w flyspray-0.9.9.5.1.new/setup: exportdb.php~
diff -ur flyspray-0.9.9.5.1/setup/index.php flyspray-0.9.9.5.1.new/setup/index.php
--- flyspray-0.9.9.5.1/setup/index.php	2008-03-07 18:55:22.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/index.php	2008-07-10 07:16:17.000000000 +0000
@@ -12,9 +12,9 @@
 ini_set('memory_limit', '64M');
 
 
-if (is_readable ('../flyspray.conf.php') && count(parse_ini_file('../flyspray.conf.php')) > 0)
+if (is_readable ('/etc/webapps/flyspray/flyspray.conf') && count(parse_ini_file('/etc/webapps/flyspray/flyspray.conf')) > 0)
 {
-   die('Flyspray already installed. Use the <a href="upgrade.php">upgrader</a> to upgrade your Flyspray, or delete flyspray.conf.php to run setup.
+   die('Flyspray already installed. Use the <a href="upgrade.php">upgrader</a> to upgrade your Flyspray, or delete /etc/webapps/flyspray/flyspray.conf to run setup.
         You can *not* use the setup on an existing database.');
 }
 
@@ -88,7 +88,7 @@
    function Setup()
    {
       // Look for ADOdb
-      $this->mAdodbPath         = APPLICATION_PATH . '/adodb/adodb.inc.php';
+      $this->mAdodbPath         = '/usr/share/php/adodb/adodb.inc.php';
       $this->mProductName       = 'Flyspray';
       $this->mMinPasswordLength	= 8;
 
@@ -125,9 +125,9 @@
       // Get the full path to the file
       $file = APPLICATION_PATH .'/' . $path;
 
-      // In case it is flyspray.conf.php, the file does not exist
+      // In case it is flyspray.conf, the file does not exist
       // so we can't tell that it is writeable. So we attempt to create an empty one
-      if ($path == 'flyspray.conf.php') {
+      if ($path == 'flyspray.conf') {
         $fp = @fopen($file, 'wb');
         @fclose($fp);
       }
@@ -305,7 +305,7 @@
                         'vars' => array(
                                     'product_name' => $this->mProductName,
                                     'message' => $this->GetPageMessage(),
-                                    'config_writeable' => $this->mWriteStatus['flyspray.conf.php'],
+                                    'config_writeable' => $this->mWriteStatus['flyspray.conf'],
                                     'config_text' => $this->mConfigText,
                                     'admin_username' => $this->mAdminUsername,
                                     'admin_password' => $this->mAdminPassword,
@@ -397,10 +397,10 @@
                                     'required_php' => $this->mPhpRequired,
                                     'php_output' => $this->CheckPhpCompatibility(),
                                     'database_output' => $this->GetDatabaseOutput(),
-                                    'config_output' => $this->CheckWriteability('flyspray.conf.php'),
+                                    'config_output' => $this->CheckWriteability('flyspray.conf'),
                                     'cache_output' => $this->CheckWriteability('cache'),
                                     'att_output' => $this->CheckWriteability('attachments'),
-                                    'config_status' => $this->mWriteStatus['flyspray.conf.php'],
+                                    'config_status' => $this->mWriteStatus['flyspray.conf'],
                                     'xmlStatus' => $this->xmlStatus,
                                     'sapiStatus' => $this->sapiStatus,
                                     'php_settings' => $this->GetPhpSettings(),
@@ -806,16 +806,16 @@
 
       $config_text = $config_intro . implode( "\n", $config );
 
-      if (is_writable('../flyspray.conf.php') && ($fp = fopen('../flyspray.conf.php', "wb")))
+      if (is_writable('/etc/webapps/flyspray/flyspray.conf') && ($fp = fopen('/etc/webapps/flyspray/flyspray.conf', "wb")))
       {
          fputs($fp, $config_text, strlen($config_text));
          fclose($fp);
-         $this->mWriteStatus['flyspray.conf.php'] = true;
+         $this->mWriteStatus['flyspray.conf'] = true;
       }
       else
       {
          $this->mConfigText = $config_text;
-         $this->mWriteStatus['flyspray.conf.php'] = false;
+         $this->mWriteStatus['flyspray.conf'] = false;
       }
 
 
Tylko w flyspray-0.9.9.5.1.new/setup: index.php~
diff -ur flyspray-0.9.9.5.1/setup/templates/complete_install.tpl flyspray-0.9.9.5.1.new/setup/templates/complete_install.tpl
--- flyspray-0.9.9.5.1/setup/templates/complete_install.tpl	2007-02-04 07:11:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/complete_install.tpl	2008-07-04 12:10:36.000000000 +0000
@@ -21,7 +21,7 @@
 							The configuration file is not writeable. You will have to upload the following
 							code manually. Click in the textarea to highlight all of the code. Copy and
 							paste the contents into the flyspray.conf.php file available in the base of
-							{$product_name} installation.
+							{$product_name} (/etc/webapps/flyspray).
 						</td>
 					</tr>
 					<tr>
@@ -37,10 +37,10 @@
 				if (!$config_writeable)
 				{
 				?>
-				<h3>flyspray.conf.php NOT writeable</h3>
+				<h3>flyspray.conf NOT writeable</h3>
 				<p>
 					To complete setup, copy and paste the contents of the textarea box into flyspray.conf.php
-					This file resides in the base of your {$product_name} installation.
+					This file resides in the base of your {$product_name} installation (/etc/webapps/flyspray).
 				</p>
 				<?php
 				}
@@ -72,4 +72,4 @@
 				</div>
 			</form>
 			</div><!-- end of right -->
-			<div class="clr"></div>
\ Brak znaku nowej linii na końcu pliku
+			<div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: complete_install.tpl~
diff -ur flyspray-0.9.9.5.1/setup/templates/license.tpl flyspray-0.9.9.5.1.new/setup/templates/license.tpl
--- flyspray-0.9.9.5.1/setup/templates/license.tpl	2007-02-04 07:11:48.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/license.tpl	2008-07-04 12:13:11.000000000 +0000
@@ -11,7 +11,7 @@
          <div class="clr"></div>
 
          <div class="formBlock" style="width:470px;position:relative;">
-            <iframe src="../docs/licences/gnu_lgpl.html" class="license" width="450" frameborder="0" scrolling="auto"></iframe>
+            <iframe src="licences/gnu_lgpl.html" class="license" width="450" frameborder="0" scrolling="auto"></iframe>
          </div>
 
          <div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: license.tpl~
diff -ur flyspray-0.9.9.5.1/setup/templates/pre_install.tpl flyspray-0.9.9.5.1.new/setup/templates/pre_install.tpl
--- flyspray-0.9.9.5.1/setup/templates/pre_install.tpl	2007-03-24 13:26:36.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/templates/pre_install.tpl	2008-07-04 12:12:30.000000000 +0000
@@ -38,7 +38,7 @@
                 </p>
                 <?php if (!$sapiStatus): ?>
                 <p><strong>CGI server API is not supported</strong>. Consider upgrading to FastCGI, otherwise you have to add
-                <code>force_baseurl = "http://yourflyspray/"</code> manually to flyspray.conf.php after setup.
+                <code>force_baseurl = "http://yourflyspray/"</code> manually to flyspray.conf after setup.
                 </p>
                 <?php endif; ?>
 			</div>
@@ -69,7 +69,7 @@
 			<div class="installBlock">
 				<table class="formBlock">
 				<tr>
-					<td valign="top">../flyspray.conf.php</td>
+					<td valign="top">../flyspray.conf</td>
 					<td align="left"><b>{!$config_output}</b></td>
 					<td>&nbsp;</td>
 				</tr>
@@ -93,12 +93,12 @@
 				</p>
 				<?php if (!$config_status): ?>
 				<p>
-				The installer has detected that the <strong>flyspray.conf.php</strong> file is not
+				The installer has detected that the <strong>flyspray.conf</strong> file is not
 				writeable. Please make it writeable by the web-server user or world writeable to
 				proceed with the setup. Alternatively if you wish to proceed, the installer will
 				make available the contents of the configuration file at the end of the setup. You
 				will then have to manually copy and paste the contents into the configuration file
-				located at <strong><?php echo APPLICATION_PATH . DIRECTORY_SEPARATOR . 'flyspray.conf.php'; ?></strong>.
+				located at <strong>'/etc/webapps/flyspray/flyspray.conf</strong>.
 				</p>
 				<?php endif; ?>
 			</div>
@@ -124,4 +124,4 @@
 			</div>
 			<div class="clr"></div>
 			</div><!-- end of right -->
-			<div class="clr"></div>
\ Brak znaku nowej linii na końcu pliku
+			<div class="clr"></div>
Tylko w flyspray-0.9.9.5.1.new/setup/templates: pre_install.tpl~
diff -ur flyspray-0.9.9.5.1/setup/upgrade.php flyspray-0.9.9.5.1.new/setup/upgrade.php
--- flyspray-0.9.9.5.1/setup/upgrade.php	2007-08-23 17:55:30.000000000 +0000
+++ flyspray-0.9.9.5.1.new/setup/upgrade.php	2008-07-10 07:16:42.000000000 +0000
@@ -39,8 +39,8 @@
 @require_once OBJECTS_PATH . '/class.tpl.php';
 
 // Initialise DB
-require_once APPLICATION_PATH . '/adodb/adodb.inc.php';
-require_once APPLICATION_PATH . '/adodb/adodb-xmlschema03.inc.php';
+require_once '/usr/share/php/adodb/adodb.inc.php';
+require_once '/usr/share/php/adodb/adodb-xmlschema03.inc.php';
 
 $db = new Database;
 $db->dbOpenFast($conf['database']);
@@ -204,7 +204,7 @@
         }
 
         $this->old_config = parse_ini_file($location, true) or die('Aborting: Could not open config file at ' . $location);
-        $this->new_config = parse_ini_file($upgrade_path . '/flyspray.conf.php', true);
+        $this->new_config = parse_ini_file($upgrade_path . '/flyspray.conf', true);
         // Now we overwrite all values of the *default* file if there is one in the existing config
         array_walk($this->new_config, array($this, '_merge_configs'));
         // save custom attachment definitions
Tylko w flyspray-0.9.9.5.1.new/setup: upgrade.php~
